pyelastix.py

```diff
@@ -301,8 +326,9 @@ def poll_process(p):
         #print("thread exit")
     
     # Start process that runs the command
-    p = subprocess.Popen(cmd, shell=True, 
-                         stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
+    p = subprocess.Popen(cmd,
```

Comment:
Have you tested on both Windows and Linux that this wont brake things?

Reply:
Tested on Linux locally, plus Linux & Windows in Azure. Seems to work without any issues.